armi/materials/thU.py

```diff
@@ -16,9 +16,6 @@
 Thorium Uranium metal.
 
 Data is from [IAEA-TECDOCT-1450]_.
-
-.. [IAEA-TECDOCT-1450] Thorium fuel cycle -- Potential benefits and challenges, IAEA-TECDOC-1450 (2005).
-    https://www-pub.iaea.org/mtcd/publications/pdf/te_1450_web.pdf
```

Comment:
I don't understand this change.

This PDF link is still valid. And it is referenced as a data source in the line above.

We can't just delete this and lose the reference.

Reply:
I think I might revert this change. Sphinx was throwing a duplicate reference warning due to this reference already being defined in thorium.py. However, this is endemic to many plugins so it is impossible to avoid all the warnings. So this change isn't improving things much.

My hope in the future is to switch to using the sphinx-bibtex extension, so all references are defined in a single place.